portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase

```diff
@@ -21,8 +21,6 @@ definition {
 		}
 		else {
 			RemoteApps.tearDown();
-
-			Page.tearDownCP();
```

Comment:
I see you also cleaned the teardown step a bit. :slightly_smiling_face:  :broom: